invenio_assets/bower.py

```diff
@@ -28,5 +43,6 @@ def __init__(self, *contents, **options):
         :param bower: bower dependencies
         :type bower: dict
         """
-        self.bower = options.pop("bower", {})
-        super(BowerBundle, self).__init__(*contents, **options)
+        raise DeprecationWarning(
+            "`Bower` has been deprecated please migrate to `npm`."
+        )
```

Comment:
Can't we just remove the bower bundle? We're still in alpha and no promises have been given, so rather get it out right away before it's released.